components/FileSelector.vue

```diff
@@ -53,7 +85,63 @@
     )
   }
 
+  async function upload_files(response_function = {}) {
+    return new Promise((resolve, reject) => {
+      for (const file of files.value) {
+        console.log(file.name)
+        const reader = new FileReader()
+        reader.onload = async function (event) {
+          const params = new FormData()
+          params.append("file", event.target.result)
+          params.append("filename", file.name)
+          params.append("filesize", file.size)
+          toggle_loading()
+          await api_fetch(
+            `${route_prefix}/upload_file`,
+            { method: "POST", body: params },
+            {
+              request_error_function: () => {
+                toggle_loading()
+                reject()
+              },
+              response_function: () => {
+                if (response_function) {
+                  response_function(response)
+                }
+                toggle_loading()
+                resolve()
+              },
+              response_error_function: () => {
+                toggle_loading()
+                reject()
+              },
+            },
+          )
+        }
+        reader.readAsDataURL(file)
+      }
+      files_uploaded.value = true
+    })
+  }
+
+  watch(files_uploaded, (value) => {
+    if (value) {
+      stepper_tree[variable_to_update] = files.value
+      stepper_tree[variable_to_increment]++
+    }
+  })
+
   onMounted(async () => {
-    await get_allowed_files()
+    if (mandatory_files.length !== 0 || additional_files.length !== 0) {
+      accept.value = "*"
+    } else {
+      await get_allowed_files()
+    }
   })
 </script>
+
+<style scoped>
+  .v-btn {
+    text-transform: unset !important;
```

Comment:
not sure about that, most of the time I like uppercase

Reply:
It seems pretty strong to me, jut my opinion